examples/bandwidthconstraints/src/index.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+var DataSeries = require('../../util/timelinegraph').DataSeries;
+var GraphView = require('../../util/timelinegraph').GraphView;
+var Prism = require('prismjs');
+var Video = require('twilio-video');
+var getRoomCredentials = require('../../util/getroomcredentials');
+var getSnippet = require('../../util/getsnippet');
+var helpers = require('./helpers');
+var waveform = require('../../util/waveform');
+var connectWithBandwidthConstraints = helpers.connectWithBandwidthConstraints;
+var updateBandwidthConstraints = helpers.updateBandwidthConstraints;
```

Comment:
Can you also use `const` and `let` instead of `var`? If `async`/`await` is supported, so is `const` and `let`.